brianlong/www.validators.app
# 183697707 - add jito stake pool

#### What's this PR do?
PR adds jito stake pool.

#### How should this be manually tested?
- execute `rails add_stake_pool:mainnet`
- `rails update_fee_in_stake_pools:mainnet`

Verify Stake Pools page in context of the jito.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183697707)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
